### ISSUE TITLE
URL not registered

### ISSUE THREAD

**Issue by andrewroberts** [Spencer-Easton/Apps-Script-Gmail-Push-Notifications#2]
When I run subscribe() I get:

```
 { "error": {
    "code": 400,
    "message": "The supplied HTTP URL is not registered in the project that owns the subscription (url=\"https://script.google.com/macros/s/AKfycbyWpY7g0XcZoAs94tuUjkCHvvjxztU504ufPOIbxda_c4ahumM/exec\", project_id=\"project-id-tdboswksprhstnxrwne\").",
    "status": "INVALID_ARGUMENT"
  }}

```

Here's [my constants](https://drive.google.com/file/d/0BxRtIprIrwuzY2xBUFNrazFWdWc/view?usp=sharing) and [the domain verification page](https://drive.google.com/file/d/0BxRtIprIrwuzbVhwWXJwM0VqWFU/view?usp=sharing). 

I can't see what the problem is so any help would be greatly appreciated.

EDIT: I've sinced tracked down the issue in the issue tracker and on stack overflow, but no clues as to why mine isn't working.

Thanks,

Andrew

**Comment by Spencer-Easton** [comment]
Are your OAuth credentials for pub/sub from the same project your domain is verified in?

**Comment by andrewroberts** [comment]
Yep, there is only the one set of credentials in the project. I've tried it
from a Apps for Work account and via the Cloub pub/sub console (
https://cloud.google.com/pubsub) where I get the same error message.

**Comment by Spencer-Easton** [comment]
I pushed a new version of this project. 
https://github.com/Spencer-Easton/Apps-Script-Gmail-Push-Notifications-v2
